### PR TITLE
docs(button): Update README

### PR DESCRIPTION
**Pull request by bonniezhou**

### CODE REVIEW

File: packages/mdc-button/README.md [hunk 1]
```diff
@@ -39,31 +35,52 @@ the `button` element. See [MDC Ripple](../mdc-ripple) and [Demo](https://materia
...
+### Styles
 
-### Adding Icon
+```scss
+@import "@material/button/mdc-button";
+```
 
-Users can nest `mdc-button__icon` inside the button element to add an icon. The icon in button
-is set to 18px to meet legibility requirements. This element should also include `aria-hidden="true"`.
+### JavaScript Instantiation
```
[the remark this answers is not in the document]
Comment (kfranqueiro) [reply packages/mdc-button/README.md]: This is an interesting thought, might want to do this for FAB too then? Currently it has its own section which we were relegating under "Additional Information" in its PR (#2648).

[the remark this answers is not in the document]
Comment (moog16) [reply packages/mdc-button/README.md]: Good idea! I'll make the same changes to be consistent.

File: packages/mdc-button/README.md [hunk 2]
```diff
@@ -39,31 +35,52 @@ the `button` element. See [MDC Ripple](../mdc-ripple) and [Demo](https://materia
...
+## Basic Usage
 
 ### HTML Structure
+
 ```html
 <button class="mdc-button">
   Button
 </button>
 ```
+> _NOTE_: Examples here use the generic `<button>`, but users can also apply the `mdc-button` class to `<a>` elements.
```
Comment (kfranqueiro) [inline packages/mdc-button/README.md]: Add newline between code block and this note

File: packages/mdc-button/README.md [hunk 3]
```diff
@@ -39,31 +35,52 @@ the `button` element. See [MDC Ripple](../mdc-ripple) and [Demo](https://materia
...
+const buttonRipple = new MDCRipple(document.querySelector('.mdc-button'));
+```
+
+> See [Importing the JS component](../../docs/importing-js.md) for more information on how to import JavaScript.
+
+## Variants
+
+### Contained Button
+
+To style a contained button, add the class `mdc-button--raised` to the `<button>` element for a contained button with elevation, or the class `mdc-button--unelevated` for a contained button flush with the surface.
```
Comment (kfranqueiro) [inline packages/mdc-button/README.md]: Can we put "class" after the class name on each of these?

File: packages/mdc-button/README.md [hunk 4]
```diff
@@ -39,31 +35,52 @@ the `button` element. See [MDC Ripple](../mdc-ripple) and [Demo](https://materia
...
 
+### Outlined Button
+
+To style an outlined button, add the class `mdc-button--outlined` to the `<button>` element.
+
+### Icons 
+
+To add an icon, add `mdc-button__icon` inside the button element and set the attribute `aria-hidden="true"`. The icon is set to 18px to meet legibility requirements.
```
Comment (kfranqueiro) [inline packages/mdc-button/README.md]: "add an element with the `mdc-button__icon` class inside the button element"

File: packages/mdc-button/README.md [hunk 5]
```diff
@@ -77,83 +94,58 @@ It's also possible to use an SVG icon:
...
-`mdc-button--unelevated` | Optional, a contained button that is flush with the surface
-`mdc-button--outlined` | Optional, a contained button that is flush with the surface and has a visible border
-`mdc-button--dense` | Optional, compresses the button text to make it slightly smaller
-
-### Disabled Button
-
-Users can add `disabled` directly to the button element or set the fieldset containing
-the button to `disabled` to disable a button. Disabled buttons cannot be interacted
-with and have no visual interaction effect.
+To disable a button, add `disabled` directly to the `<button>`, or set `disabled` on the `<fieldset>` containing the button.
```
Comment (kfranqueiro) [inline packages/mdc-button/README.md]: "add the `disabled` attribute" / "or set the `disabled` attribute"

**Pull request merged**